pkg/helmt/helmt.go

```diff
@@ -130,18 +131,38 @@ func template(name string, chart string, values []string, namespace string, skip
 	return execute("helm", execOpts{}, args...)
 }
 
-func fetch(repository, chart, version string, username, password string) error {
+func fetch(repository, chart, version string, username, password string) (string, error) {
+	d, err := tempDir()
+	if err != nil {
+		return "", fmt.Errorf("failed to create temporary directory to download helm chart: %v", err)
+	}
+
 	args := []string{"fetch"}
 	args = append(args, "--repo", repository)
 	args = append(args, "--version", version)
+	args = append(args, "--destination", d)
 	if username != "" {
 		args = append(args, "--username", username)
 	}
 	if password != "" {
 		args = append(args, "--password", password)
 	}
 	args = append(args, chart)
-	return execute("helm", execOpts{}, args...)
+	err = execute("helm", execOpts{}, args...)
```

Comment:
From here to line 163 one could extract a method which just returns the filename of the downloaded chart